.travis.yml

```diff
@@ -1,34 +1,43 @@
 language: julia
+
 os:
   - linux
+  - osx
+  - windows
```

Comment:
the thinking in only testing Gadfly on linux is that all the OS-specific code is in Compose.  no need to burden the Travis servers.

Reply:
I did think that to be the case. Though their brief inclusion was useful to see Gadfly working with the new binary provided Cairo. Anyhow, I've removed macOS and Windows from the test matrix.